rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -370,7 +370,7 @@ http {
     {{ end }}
 
     {{ if not $all.DynamicConfigurationEnabled }}
-    {{ range $name, $upstream := $backends }}
+    {{ range _, $upstream := $backends }}
```

Comment:
Go templates do not support this syntax, you must either use both `$index` and `$value`, or just `$value`.
My fault, my previous comment was misleading,